docs/api/action-sheet.md

```diff
@@ -94,10 +94,14 @@ import CssCustomProperties from '@site/static/usage/v7/action-sheet/theming/css-
 
 ## Accessibility
 
+### Screen Readers
+
 Action Sheets are given a `role` of [`dialog`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role). In order to align with the ARIA spec, either the `aria-label` or `aria-labelledby` attribute must be set.
 
 It is strongly recommended that every Action Sheet have the `header` property defined, as Ionic will automatically set `aria-labelledby` to point to the header element. However, if you choose not to include a `header`, an alternative is to use the `htmlAttributes` property to provide a descriptive `aria-label` or set a custom `aria-labelledby` value.
 
+Buttons containing text will be read by a screen reader. If a button contains only an icon, a label should be assigned to the button by passing `aria-label` or `aria-labelledby` to the `htmlAttributes` property. The `aria-label` property accepts a text description and `aria-labelledby` accepts an element that contains the description of the button.
```

Comment:
I think we could perhaps make this slightly more clear that they need to set the `htmlAttributes` property on the button, not on the action sheet itself. The rest of this section refers to the action sheet as a whole, and the paragraph before this one even includes a reference to `htmlAttributes` on the action sheet. A code snippet would be nice, but more specific text would suffice.

Reply:
Separated them out into different subheadings and added code examples to each. Lmk if that's better!

Reply:
This is great! Thanks! 💯